usaspending_api/reporting/management/sql/populate_reporting_agency_tas.sql

```diff
@@ -19,8 +19,11 @@ FROM (
             treasury_account_identifier AS tas_id,
             SUM(obligations_incurred_total_by_tas_cpe) AS appropriation_obligated_amount
         FROM appropriation_account_balances AS aab
-        JOIN submission_attributes AS sa
+        INNER JOIN submission_attributes AS sa
```

Comment:
thanks for the inner joins

Reply:
Most recent runs (after the commit immediately below) showed a slight increase to about 20 seconds for the two SQL files, but overall still very quick.